packages/main/src/ListItem.ts

```diff
@@ -481,6 +482,7 @@ abstract class ListItem extends ListItemBase {
 			ariaHaspopup: this.accessibilityAttributes.hasPopup,
 			setsize: this.accessibilityAttributes.ariaSetsize,
 			posinset: this.accessibilityAttributes.ariaPosinset,
+			listItemAriaLabel: this.typeActive && document.activeElement === this ? ListItem.i18nBundle.getText(LIST_ITEM_ACTIVE) : undefined,
```

Comment:
why we need to set it only when the active element is the current list item?

Reply:
Is it going to be read out by the screen readers if always render that text?

I'm thinking about case where we have button inside list item and that button is focused? Also similar case is when we there is selection mode with selection component.